tests/Behat/Content/Node/NewsContentContext.php

```diff
@@ -40,6 +40,7 @@ public function alterNewsFields(BeforeParseEntityFieldsScope $scope): void {
       'Reference' => 'oe_reference_code',
       'Location' => 'oe_news_location',
       'Body text' => 'body',
+      'Sources' => 'oe_news_sources',
```

Comment:
Do we need to add mapping for "Persons" and "Last updated on" fields for further usage?

Reply:
Added the `Last updated date` field but since in oe_content we only provide the storage for the persons field and we can't control the field label it would not help for any third-party site.

Reply:
This Behat context is used to provide data for News CT so it will be helpful.